app/assets/stylesheets/application.scss

```diff
@@ -173,19 +173,20 @@ body {
       h1 {
         margin-bottom: 1.5rem;
 
-        &.diy-header {
+        &.landing-page-header {
           margin-top: 0;
         }
       }
     }
 
-    .diy-header {
+    .landing-page-header {
       max-width: 450px;
     }
 
-    .diy-subheader {
+    .landing-page-subheader {
       font-weight: 200;
       max-width: 450px;
+      position: relative;
```

Comment:
the way the hero image was implemented, links in text next to it were unclickable. we just didn't have any links before.

Reply:
Yikes :) Good catch.

Personally I'd prefer to understand this better, but it's not a blocker for merging this. I'm hopeful we can use `pointer-events: all;` instead.

Feel free to merge as-is, and maybe I'll play with that.

Reply:
yeah it is a little mysterious, appreciate that!